build.sbt

```diff
@@ -37,6 +37,11 @@ inThisBuild(Seq[Setting[_]](
   )
 ))
 
+ThisBuild / publishMavenStyle      := true
```

Comment:
This had to be done this way because of

```
Reference to undefined setting: 

  ThisBuild / sonatypePublishToBundle from ThisBuild / publishTo (/Users/mdedetrich/github/jardiff/build.sbt:3)
     Did you mean core / sonatypePublishToBundle ?
```

At some point it may be better to just use `ThisBuild / ` for everything but thats more a nicety